assets/js/modules/analytics/components/setup/SetupFormUA.js

```diff
@@ -16,24 +16,78 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import { useMount } from 'react-use';
+
 /**
  * WordPress dependencies
  */
+import { __ } from '@wordpress/i18n';
 import { Fragment } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
-import { AccountSelect } from '../common';
+import Data from 'googlesitekit-data';
+import {
+	AccountSelect,
+	ExistingGTMPropertyNotice,
+	ExistingTagNotice,
+	ProfileSelect,
+	PropertySelect,
+	ProfileNameTextField,
+} from '../common';
+import { STORE_NAME, PROFILE_CREATE } from '../../datastore/constants';
+import { MODULES_ANALYTICS_4, PROPERTY_CREATE } from '../../../analytics-4/datastore/constants';
+import GA4PropertyNotice from '../common/GA4PropertyNotice';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import { isValidPropertyID } from '../../util';
+
+const { useSelect, useDispatch } = Data;
 
 export default function SetupFormUA() {
+	const { selectProperty } = useDispatch( MODULES_ANALYTICS_4 );
+	const accounts = useSelect( ( select ) => select( STORE_NAME ).getAccounts() ) || [];
+	const hasExistingTag = useSelect( ( select ) => select( STORE_NAME ).hasExistingTag() );
+	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+
+	// Needed to conditionally show the profile name field and surrounding container.
+	const profileID = useSelect( ( select ) => select( STORE_NAME ).getProfileID() );
+
+	useMount( () => {
+		selectProperty( PROPERTY_CREATE );
+	} );
+
 	return (
 		<Fragment>
+			<StoreErrorNotices moduleSlug="analytics" storeName={ STORE_NAME } />
+
+			<ExistingTagNotice />
+			{ ! hasExistingTag && <ExistingGTMPropertyNotice /> }
+
+			{ ( !! accounts.length && ! hasExistingTag ) && (
+				<p className="googlesitekit-margin-bottom-0">
+					{ __( 'Please select the account information below. You can change this view later in your settings.', 'google-site-kit' ) }
+				</p>
+			) }
+
 			<div className="googlesitekit-setup-module__inputs">
 				<AccountSelect />
+
+				<PropertySelect />
+
+				{ isValidPropertyID( propertyID ) && <ProfileSelect /> }
 			</div>
-			<div>SetupFormUA</div>
+
+			{ profileID === PROFILE_CREATE && (
+				<div className="googlesitekit-setup-module__inputs googlesitekit-setup-module__inputs--multiline">
+					<ProfileNameTextField />
+				</div>
+			) }
+
+			{ isValidPropertyID( propertyID ) && <GA4PropertyNotice /> }
```

Comment:
@eugene-manuilov The doc here: https://docs.google.com/document/d/11DMYyAGhtFHO8bMH107pacC8iMpD9PJ9BTaR6IF_B5E/edit?resourcekey=0-0CgoW31PILS2t6JGZ6w_yA#heading=h.6t1gq9q9bzg3 states that 
> Once a property has been selected, the user sees the view dropdown, as well as a notice below that informs them that an associated GA4 property will also be created.

So the notice is not visible when the component is rendered.

Reply:
It doesn't matter which property you select, a new GA4 property will be created anyway because the currently selected account doesn't have GA4 properties at all.

Reply:
I see. Updated